dev-python/python-rapidjson/python-rapidjson-1.20.ebuild

```diff
@@ -0,0 +1,26 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_EXT=1
+inherit distutils-r1 pypi
+
+DESCRIPTION="Python wrapper around rapidjson"
+HOMEPAGE="
+	https://github.com/python-rapidjson/python-rapidjson
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/pysimdjson-5.0.1[${PYTHON_USEDEP}]
+	>=dev-python/orjson-3.7.6[${PYTHON_USEDEP}]
+	>=dev-python/simplejson-3.17.6[${PYTHON_USEDEP}]
+	>=dev-python/ujson-5.4.0[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+# too much benchmarking
+#distutils_enable_tests pytest
```

Comment:
Use `RESTRICT=test` instead, that way the ebuild retains some test support since the restriction is overridable at runtime.